lib/osf-components/addon/components/contributors/card/readonly/template.hbs

```diff
@@ -0,0 +1,45 @@
+<div data-test-contributor-card ocal-class='CardContainer'>
+    <div data-test-contributor-card-main local-class='MainSection'>
+        <span local-class='CardSection'>
+            <img
+                data-test-contributor-gravatar
+                class='m-l-xs'
+                src={{@contributor.users.links.profile_image}}
+                alt={{@contributor.users.fullname}}
+                height='30'
+                width='30'
+            >
+            <OsfLink
+                data-test-contributor-link
+                data-analytics-name='View user'
+                @route='guid-user'
+                @models={{array @contributor.id}}
+            >
+                {{@contributor.fullName}}
+            </OsfLink>
+        </span>
+        <span data-test-contributor-permission local-class='CardSection'>
+            {{@contributor.permission}}
+        </span>
+        <span data-test-contributor-citation local-class='CardSection'>
+            {{@contributor.bibliographic}}
+            <OsfButton
+                data-test-expand-user-card
+                data-analytics-name='Expand contributor card'
+                @type='link'
+                @onClick={{action this.toggleDropdown}}
+            >
+                <i class={{if this.showExpandedList 'fa fa-angle-up' 'fa fa-angle-down'}}></i>
+            </OsfButton>
+        </span>
+    </div>
+    {{#if this.showExpandedList}}
```

Comment:
Nitpick: if `showExpandedList` is used to determine employer info, then maybe `showEmployerInfo` or sth. similar would be better.